hydrolib/core/dflowfm/extold/__init__.py

```diff
@@ -0,0 +1,10 @@
+from ..extold.models import ExtForcing, ExtOldModel
```

Comment:
Is there any specific reason you import it like that instead of simply:
`from .models import ExtForcing, ExtOldModel`

Reply:
Nope, not sure why I did it like that. Fixed it.